src/openrct2/scenario/Scenario.cpp

```diff
@@ -237,8 +237,8 @@ void scenario_success_submit_name(const char* name)
  */
 static void scenario_entrance_fee_too_high_check()
 {
-    money16 totalRideValueForMoney = gTotalRideValueForMoney;
-    money16 max_fee = totalRideValueForMoney + (totalRideValueForMoney / 2);
+    const money16 totalRideValueForMoney = gTotalRideValueForMoney;
```

Comment:
You can remove this intermediate variable altogether. It’s most likely there since the times of RCT_GLOBAL(), which ended in October 2016 ;)